ranch/ranch.go

```diff
@@ -410,25 +404,6 @@ func (r *Ranch) SyncConfig(configPath string) error {
 	return r.Storage.SyncResources(config)
 }
 
-// StartDynamicResourceUpdater starts a goroutine which periodically
-// updates all dynamic resources.
-func (r *Ranch) StartDynamicResourceUpdater(updatePeriod time.Duration) {
```

Comment:
Am I understanding correctly that the reason we no longer need this is because we'll basically be syncing all resources anytime one of the dynamic resources is created or deleted?

(This code was added in https://github.com/kubernetes/test-infra/issues/15906, and I'm trying to convince myself that it still does the appropriate thing. I guess the unit tests help enforce that.)

Reply:
> Am I understanding correctly that the reason we no longer need this is because we'll basically be syncing all resources anytime one of the dynamic resources is created or deleted?

Yes, anytime any of the dynamic resources changes in any way, the config changes or resources get set to tombstone state